lms/djangoapps/verify_student/management/commands/set_software_secure_status.py

```diff
@@ -0,0 +1,58 @@
+"""
+Manually set Software Secure verification status.
+"""
+
+import sys
+
+from django.core.management.base import BaseCommand
+from optparse import make_option
+from verify_student.models import SoftwareSecurePhotoVerification, VerificationCheckpoint
+
+
+class Command(BaseCommand):
+    """
+    
+    """
+
+    args = "<{approved, denied}, SoftwareSecurePhotoVerification id, [reason_for_denial]>"
+
+    def handle(self, *args, **kwargs):
+        from verify_student.views import _set_user_requirement_status
+
+        status_to_set = args[0]
+        receipt_id = args[1]
+
+        try:
+            attempt = SoftwareSecurePhotoVerification.objects.get(receipt_id=receipt_id)
+        except SoftwareSecurePhotoVerification.DoesNotExist:
+            self.stderr.write(
+                'SoftwareSecurePhotoVerification with id {id} could not be found.\n'.format(id=receipt_id)
+            )
+            sys.exit(1)
+
+        if status_to_set == 'approved':
+            self.stdout.write('Approving verification for {id}.\n'.format(id=receipt_id))
+            attempt.approve()
+            _set_user_requirement_status(attempt, 'reverification', 'satisfied')
+
+        elif status_to_set == 'denied':
+            self.stdout.write('Denying verification for {id}.\n'.format(id=receipt_id))
+            if len(args) >= 3:
+                reason_for_denial = args[2]
+            else:
+                reason_for_denial = 'Denied via management command.'
+            attempt.deny(reason_for_denial)
+            _set_user_requirement_status(attempt, 'reverification', 'failed', reason_for_denial)
```

Comment:
I think `_set_user_requirement_status` takes the fourth argument as json value like the way implemented here: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/verify_student/views.py#L1298